src/Client.php

```diff
@@ -227,18 +245,33 @@ protected function restCall($apiUrl, $verb, $guzzleParams, $apiversion='',$heade
             $JsonBody = json_decode($body);
             if (is_null($JsonBody)) {
                 if (json_last_error() === JSON_ERROR_SYNTAX) {
-				     throw new \Exception('The response is in unexpected format. The response is: ' . $JsonBody);
-			     }
-              return $body;
+                    $errorMsg = 'The response is in unexpected format. The response is: ';
+                    // populate exception details in log object
+                    $logModel-> populateErrorInfoWithMessageAndBody($errorMsg, $response);
+				    throw new \Exception($errorMsg . $JsonBody);
```

Comment:
nit: tabbing

Reply:
This looks fine in the code though, not sure why it is being displayed like that on UI.
![Screen Shot 2022-11-02 at 11 31 56 AM](https://user-images.githubusercontent.com/59040125/199572883-9365c083-01ad-4d1b-ac8e-83e994e13c08.png)